code/controlconfig/controlsconfig.cpp

```diff
@@ -393,21 +393,10 @@ int control_config_detect_axis()
 
 void control_config_conflict_check()
 {
-	int i, j, shift = -1, alt = -1;
-
+	int i, j;
+	
 	for (i=0; i<CCFG_MAX; i++) {
 		Conflicts[i].key = Conflicts[i].joy = -1;
-		switch (Control_config[i].key_id) {
-			case KEY_LSHIFT:
-			case KEY_RSHIFT:
-				shift = i;
```

Comment:
There is some similar alt/shift logic in controlsconfigcommon.cpp, and in that file the shift (and probably alt) values are checked later in the code.  Is there a chance these values are used there, shared between these files?